packages/fiori-generator-shared/src/npm-package-scripts/getPackageScripts.ts

```diff
@@ -77,21 +63,18 @@ function buildStartCommand(localOnly: boolean, params: string, startFile?: strin
 /**
  * Generates a variant management script in preview mode.
  *
- * @param {string} sapClient - The SAP client parameter to include in the URL. If not provided, the URL will not include the `sap-client` parameter.
+ * @param {boolean} addSearchParams - Indicates whether to include search parameters in the command URL.
```

Comment:
Can we add a specific description as to what gets added and why, as search params are not being passed in, in this case.